site/docs/hosting/cloudflare-workers.md

```diff
@@ -52,9 +52,9 @@ Edit `src/index.js` or `src/index.ts`, and write this code inside:
 // Note that we're importing from 'grammy/web', not 'grammy'.
 import { Bot, webhookCallback } from "grammy/web";
 
-// You can replace `BOT_TOKEN` with your bot token, but it is better to store it in an environment variable.
-// For more on this, see https://developers.cloudflare.com/workers/platform/environment-variables/#secrets-on-deployed-workers.
-// You should also replace `BOT_INFO` with your bot info obtained from `bot.api.getMe()`.
+// The following line of code assumes that you have configured the secrets BOT_TOKEN and BOT_INFO.
```

Comment:
Should we tell them how to access these values, rather than just using an undeclared variable?

Reply:
It is going to be declared as a global variable in runtime.

Reply:
Oof.